app/views/posts/index.html.erb

```diff
@@ -1 +1,26 @@
-<h1>Here is a list of posts for a given user</h1>
+<section class="w-full flex items-center gap-2 border-2 p-4">
+  <img class="w-24 h-24" src=<%= User.find(params[:user_id]).photo %> alt=<%= User.find(params[:user_id]).name %> />
+  <div>
+  <p>Name: <%= User.find(params[:user_id]).name %></p>
+  <p>Number of posts: <%= User.find(params[:user_id]).posts_counter %></p>
+  </div>
+</section>
+<ul class="w-full flex flex-col gap-2 p-4">
+  <% @posts.each do |post| %>
+    <li class="border-2 p-2">
+      <div>
+        <p class="text-md font-semibold">Post #<%= post.id %>: <%= post.title %></p>
+        <p><%= post.text %></p>
+        <div class="w-3/6 flex items-center gap-8">
+          <p>Comments: <%= post.comments_counter %></p>
```

Comment:
- [ ] Comment number is showing wrong. Kİndly fix it.

![image](https://user-images.githubusercontent.com/38635158/185726395-f6d3f523-ecda-4cdc-af86-d5af54e83b29.png)
![image](https://user-images.githubusercontent.com/38635158/185726405-6e8421e4-5cab-462c-a3e0-2ee5bb70b926.png)

Reply:
Thanks, it's done!!

Reply:
Done!!